Patch-Source: https://dovecot.org/mailman3/archives/list/dovecot@dovecot.org/thread/MFHSI5N6IEHN7TBIHAY3IUY7QIF4GAU5/#MFHSI5N6IEHN7TBIHAY3IUY7QIF4GAU5

Update setting types to fix build error on 32-bit arches.
---
--- a/src/lib-http/http-server.h
+++ b/src/lib-http/http-server.h
@@ -49,8 +49,8 @@
 	/* The kernel send/receive buffer sizes used for the connection sockets.
 	   Configuring this is mainly useful for the test suite. The kernel
 	   defaults are used when these settings are 0. */
-	size_t socket_send_buffer_size;
-	size_t socket_recv_buffer_size;
+  uoff_t socket_send_buffer_size;
+  uoff_t socket_recv_buffer_size;
 };
